### PR TITLE
SNOW-1527724: bump up awssdk arrow build version

### PR DESCRIPTION
**Pull request by sfc-gh-ext-simba-hx**
made changes in #731 but forgot to bump up build version.
bump up build version of awssdk and arrow to have them rebuilt and uploaded to s3.

### CODE REVIEW

**Review by sfc-gh-dstempniak — APPROVED** [review]

File: scripts/build_awssdk.bat [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ::
 @echo off
 set aws_src_version=1.11.283
-set aws_build_version=6
+set aws_build_version=7
```
Comment (sfc-gh-dprzybysz) [inline scripts/build_awssdk.bat:7]: windows 32 build fails for vs14 with:
```
2024-07-19 01:13:14  deps/aws-sdk-cpp-1.11.283/src/aws-cpp-sdk-core/source/client/RequestCompression.cpp(89): error C2220: warning treated as error - no 'object' file generated (compiling source file deps\aws-sdk-cpp-1.11.283\cmake-build-win32-VS14-Release\src\aws-cpp-sdk-core\ub_core.cpp) [deps\aws-sdk-cpp-1.11.283\cmake-build-win32-VS14-Release\src\aws-cpp-sdk-core\aws-cpp-sdk-core.vcxproj]
```
and for vs 17
```
2024-07-19 01:12:16  deps\aws-sdk-cpp-1.11.283\src\aws-cpp-sdk-core\source\client\RequestCompression.cpp(89,27): error C2220: the following warning is treated as an error [deps\aws-sdk-cpp-1.11.283\cmake-build-win32-VS17-Release\src\aws-cpp-sdk-core\aws-cpp-sdk-core.vcxproj]
2024-07-19 01:12:16    (compiling source file 'ub_core.cpp')
```

Comment (sfc-gh-dstempniak) [reply scripts/build_awssdk.bat:7]: It failed only on Win32. 
Win64, Linux and Mac are ok.

Comment (sfc-gh-ext-simba-hx) [reply scripts/build_awssdk.bat:7]: @sfc-gh-dprzybysz @sfc-gh-dstempniak 
Tried to add some fix in `RequestCompression.cpp` but seems not working. I can't reproduce the issue with either vs14 or vs17, and vs16 build on github also works.
Please continue to share the error information. In the worst case we could revert the build flag changes for awssdk made in #731 to unblock the release. That's for binskim check which doesn't have functionality impact.

Comment (SimbaGithub) [reply scripts/build_awssdk.bat:7]: Thanks, issue fixed.

**Review by sfc-gh-dprzybysz — APPROVED** [review]

**Pull request merged**